apis/core/bootstrap/shapes/dataset.ts

```diff
@@ -189,19 +189,25 @@ ${shapeId} {
     ${sh.property} [
       ${sh.name} "Creation Date" ;
```

Comment:
If we're going to keep these dates around in the form, we need to add some `sh:description` to them.

Reply:
Ok, TBH I'm not sure what those should be